pkg/eventemitter/eventemitter.go

```diff
@@ -65,33 +66,51 @@ func (ee eventEmitter) EmitEvent(object runtime.Object, eventType, reason, msg s
 }
 
 func (ee eventEmitter) EmitProgressingForConfig() {
-	config := ee.getConfigForEmitter()
+	config, err := ee.getConfigForEmitter()
+	if err != nil {
+		log.Printf(err.Error())
```

Comment:
I would like to see a more specific error message so that a debugger will know which emmiter function failed
(just a suggestion, but the important thing is the "**Progressing**") `log.Printf("Failed to emit event Progressing for config. err = %v", err)`

Reply:
Done.